frontend/package.json

```diff
@@ -2,6 +2,18 @@
   "name": "hello-world-app",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
```

Comment:
As we decided for the cypress runner, can we just remove the Jest test runner with scripts alltogether?